sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -157,28 +157,28 @@ class SparkSessionBuilderSuite extends SparkFunSuite with BeforeAndAfterEach {
     "spark context conf values in SessionState") {
     val session = SparkSession.builder()
       .master("local")
-      .config(GLOBAL_TEMP_DATABASE.key, value = "globalTempDB-SPARK-31234")
+      .config(GLOBAL_TEMP_DATABASE.key, value = "globaltempdb-spark-31234")
```

Comment:
Now, I understand why you change the conf. +1.

Reply:
Need this one? Dropping this looks fine to me though.